test/test_shared.cpp

```diff
@@ -3061,4 +3061,25 @@ TEST(Shared_Bptree_insert_failure)
     g.get_table(0)->add_empty_row(396);
 }
 
+TEST(SharedGroupOptions_tmp_dir)
```

Comment:
If we go this way please use NONCONCURRENT_TEST as the static could have side effects on other tests.